src/convertToObject.js

```diff
@@ -6,7 +8,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (typeof sourceString !== 'string') {
+    throw new TypeError('Input must be a string.');
+  }
+
+  return sourceString
```

Comment:
There is a missing semicolon at the end of this line. Adding a semicolon here will prevent potential syntax errors.